src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/TagHelperTooltipFactory.cs

```diff
@@ -11,5 +11,9 @@ internal abstract class TagHelperTooltipFactory
         public abstract bool TryCreateTooltip(AggregateBoundElementDescription descriptionInfos, out MarkupContent markupContent);
 
         public abstract bool TryCreateTooltip(AggregateBoundAttributeDescription descriptionInfos, out MarkupContent markupContent);
+
+        public abstract bool TryCreateTooltip(AggregateBoundElementDescription descriptionInfos, out RazorClassifiedTextElement classifiedTextElement);
```

Comment:
Hmmm, for another day we may want to consider refactoring all the VS pieces of this into its own `VSLSPTagHelperTooltipFactory` and `LSPTagHelperTooltipFactory`. Don't have to do it now but food for thought.